addons/binding/org.openhab.binding.mihome/ESH-INF/thing/channel.xml

```diff
@@ -188,6 +188,13 @@
 		<state readOnly="true"></state>
 	</channel-type>
 
+    <channel-type id="lastAction">
```

Comment:
The xml files do not look okay yet. Can you please have a second look?

Reply:
fixed

Reply:
...and rebased